python/lsst/meas/algorithms/simple_curve.py

```diff
@@ -97,9 +97,16 @@ def evaluate(self, detector, position, wavelength, kind='linear'):
             The position on the detector at which to evaluate the curve.
         wavelength : `astropy.units.Quantity`
             The wavelength(s) at which to make the interpolation.
-        kind : `str`
-            The type of interpolation to do.  See documentation for
-            `scipy.interpolate.interp1d` for accepted values.
+        kind : `str`, optional
+            The type of interpolation to do (default is 'linear').
+            See documentation for `scipy.interpolate.interp1d` for
+            accepted values.
+        bounds_error : `bool`, optional
+            Raise error if interpolating outside the range of x
+            (default is False)?
```

Comment:
I'd put the question mark at the end of the line above and have the default state as a distinct item. Otherwise I keep thinking I'm being asked what the default is. Also add a Raises section to the docstring saying what this exception might be.